src/fides/api/ops/service/messaging/message_dispatch_service.py

```diff
@@ -138,7 +138,7 @@ def dispatch_message(
         raise MessageDispatchException("No notification service type configured.")
 
     logger.info("Retrieving message config")
-    messaging_config: MessagingConfig = MessagingConfig.get_configuration(
+    messaging_config = MessagingConfig.get_configuration(
```

Comment:
Noting a type hint being removed likely to get around linting issues

Reply:
yep....I also have no idea why, but I had to add _a lot_ of mypy workarounds and that makes me concerned, but not concerned enough to spend hours figuring out why 😅